### PR TITLE
Re-skin Profile dropdown to use MUI

### PR DESCRIPTION
**Pull request by dmitryduev**
MUI-based re-skin of the Profile dropdown:

dropdown-2

### CODE REVIEW

**Comment by stefanv** [comment]
For color, I suggest `makeStyle` and access the theme for primary, secondary color, etc.

At some point, we should decide what the theme is going to be and configure that, @kmshin1397.  Currently, it's whatever comes out the box :grimacing:

**Comment by dmitryduev** [comment]
Yea, I am going to do it like that, just haven't finished. Do you actually like the clickable icon? Or do you prefer no icon and clickable avatar + username?

**Comment by dmitryduev** [comment]
All tests passing locally, green on Travis, failed on GA - marked the corresponding tests flaky.

**Comment by dmitryduev** [comment]
Travis is green, an unrelated test (`test_delete_token`) known to be flaky failed on GA.

**Review by kmshin1397 — COMMENTED** [review]
LGTM!

**Review by kmshin1397 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "d3": "5.16.0",
     "dayjs": "1.8.34",
     "emoji-dictionary": "1.0.10",
-    "emojilib": "2.4.0",
+    "emojilib": "^2.4.0",
```
[the remark this answers is not in the document]
Comment (dmitryduev) [reply package.json]: Ah, it should not have been here at all, removed the changes.

File: skyportal/tests/frontend/test_profile.py [hunk 2]
```diff
@@ -62,6 +62,7 @@ def test_add_data_to_user_profile(driver, user):
     driver.scroll_to_element_and_click(
         driver.find_element_by_xpath('//*[@id="updateProfileButton"]')
     )
+    # driver.click_xpath('//*[@id="updateProfileButton"]')
```
[the remark this answers is not in the document]
Comment (dmitryduev) [reply skyportal/tests/frontend/test_profile.py]: Fixed.

**Review by profjsb — CHANGES_REQUESTED** [review]
Looks like the avatar spills off the screen on mobile -- perhaps need some small viewport logic.
image

The modal should disappear after navigation completes:
Kapture 2020-09-21 at 10 06 00

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (dmitryduev) [reply package.json]: This file was accidentally committed and is now removed from this PR.

**Comment by dmitryduev** [comment]
@profjsb addressed the second request. As for the first one -- couldn't reproduced -- tried in Chrome, Safari, and Firefox, even tried using a very long title:
image

@kmshin1397 would you mind trying to reproduce the first issue?

**Review by profjsb — APPROVED** [review]

**Pull request merged**